test/e2e/bind/happy-case_test.go

```diff
@@ -36,205 +36,318 @@ import (
 	"sigs.k8s.io/yaml"
 
 	kubebindv1alpha1 "github.com/kube-bind/kube-bind/pkg/apis/kubebind/v1alpha1"
+	clusterscoped "github.com/kube-bind/kube-bind/pkg/konnector/controllers/cluster/serviceexport/cluster-scoped"
 	providerfixtures "github.com/kube-bind/kube-bind/test/e2e/bind/fixtures/provider"
 	"github.com/kube-bind/kube-bind/test/e2e/framework"
 )
 
 func TestClusterScoped(t *testing.T) {
 	t.Parallel()
 
-	testHappyCase(t, kubebindv1alpha1.ClusterScope)
+	// cluster scoped resource, with cluster scoped informers
+	testHappyCase(t, kubebindv1alpha1.ClusterScope, kubebindv1alpha1.ClusterScope)
 }
 
 func TestNamespacedScoped(t *testing.T) {
 	t.Parallel()
 
-	testHappyCase(t, kubebindv1alpha1.NamespacedScope)
+	// namespaced resource, with namespace scoped informers
+	testHappyCase(t, kubebindv1alpha1.NamespacedScope, kubebindv1alpha1.NamespacedScope)
+	// namespaced resource, but with cluster scoped informers
+	testHappyCase(t, kubebindv1alpha1.NamespacedScope, kubebindv1alpha1.ClusterScope)
 }
 
-func testHappyCase(t *testing.T, scope kubebindv1alpha1.Scope) {
+func testHappyCase(t *testing.T, resourceScope, informerScope kubebindv1alpha1.Scope) {
```

Comment:
nit (follow-up is fine): would use `ResourceScope` from apiextension-apiserver for the first https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types.go#L255

Reply:
Thanks for the suggestion! Just did this in #180.